docs/src/main/paradox/elasticsearch.md

```diff
@@ -24,20 +24,21 @@ The table below shows direct dependencies of this module and the second tab show
 @@dependencies { projectId="elasticsearch" }
 
 
-## Set up REST client
+## ElasticSearch Parameters
```

Comment:
I believe starting with the connection settings feels more natural and is the proper replacement for the REST client.

The branded spelling does surprisingly use a lower-case `s`:
```suggestion
## Elasticsearch Parameters
```